packages/tools/client-debugger/client-debugger-view/src/Menu.tsx

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { DefaultPalette, IStackStyles, Stack } from "@fluentui/react";
+import React from "react";
+
+/**
+ * Props for {@link MenuSection}
+ */
+export interface MenuSectionProps {
+	header: string;
+}
+
+/**
+ * Generic component for a section of the menu.
+ *
+ * @internal
+ */
+export function MenuSection(props: React.PropsWithChildren<MenuSectionProps>): React.ReactElement {
+	const { header, children } = props;
+
+	return (
+		<Stack styles={menuSectionStyles}>
+			<Stack.Item styles={menuSectionHeaderStyles}>{header}</Stack.Item>
+			{children}
+		</Stack>
+	);
+}
+
+/**
+ * Props for {@link MenuItem}
+ */
+export interface MenuItemProps {
+	onClick: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
Nit: Unless we expect the consumers of `MenuItem` to need the pointer event metadata, we can probably simplify the consumer side of this by simply using `() => void` here instead.